doc/1-intro/faq.md

```diff
@@ -0,0 +1,138 @@
+# frequently asked questions
+
+## general
+
+### how do I use Furnace?
+
+the [quick start guide](quickstart.md) and [concepts list](concepts.md) are great first steps in getting used to the tracker way. after that, try exploring the demo songs to learn more about how to get the sounds you want.
+
+### is there an Android build?
+
+yes and no. Furnace has been successfully built for Android and it even has some rudimentary touch UI support, but it's extremely unfinished and extremely unsupported. there are many good reasons that it's not in the official releases yet.
+
+### is there an iOS build?
+
+nope.
+
+### will Furnace ever have a piano roll or DAW interface?
+
+there are no plans for this.
+
+### I've lost my song – what do I do?
+
+Furnace keeps backups of the songs you've worked on before. go to **file > restore backup**.
+
+## tracking
+
+### how do I compose in 6/8, 5/4, or other time signatures?
+
+set the pattern length to a multiple of the number of beats (6 or 5 as mentioned above). don't forget to change the row highlight values to match!
+
+### how do I track triplets, quintuplets, or other tuplets?
+
+there are two common methods:
+- use delay commands (`EDxx`) to offset notes into their correct places. this is good for the occasional set of tuplets, but if you expect to use a lot of them...
+- plan ahead for the song to have them by making your pattern length a multiple of that number. remember to adjust row highlight values to match.
+
+depending on the tempo of the song, it may only be possible to get perfectly even tuplets by changing the tick rate. mind that the result may not be authentic to hardware.
+
+### why do certain notes not play low enough, high enough, or in tune?
+
+each chip has its own set of limitations regarding what frequencies it can play. if these limits are likely to be found in normal tracking, they'll be mentioned in [that chip's documentation](../7-systems/).
+
+### does Furnace support tunings other than twelve tone equal temperament?
```

Comment:
Not sure whether we should mention this one... it might draw attention to..... oh no don't make me remember

Reply:
It's come up a couple times and I figure it's a safe entry. Folks who don't care will gloss right over it, and folks who do care will be really glad they didn't have to pop onto the Discord just to ask about it. If you want it purged, though, I can totally do that. I just need a definitive answer. :)

Reply:
Yeah. Please remove it.